src/random_bq_to_anndata.py

```diff
@@ -0,0 +1,46 @@
+from google.cloud import bigquery
+import argparse
+import random
+import anndata as ad
+
+
+def get_random_ids(project, dataset, num_cells):
+    client = bigquery.Client()
```

Comment:
Can we create the client in one spot and pass it in to these two functions? Also George and I have sometimes found a need for the project to be explicitly specified a la

```
client = bigquery.Client(project=project)
```